tests/components/soundtouch/test_media_player.py

```diff
@@ -29,6 +36,11 @@ class MockDevice(STD):
     def __init__(self):
         """Init the class."""
         self._config = MockConfig
+        self._zone_status = MockZoneStatus
```

Comment:
thanks @tadly - but as I wrote on Slack - you didn't have to review it. Will give it one last try then. Wondered myself why the class `MockConfig` is not instantiated, but since I'm not a Python dev I didn't question it.